src/backend/TrafficCourts/Workflow.Service/Services/OracleDataApiService.cs

```diff
@@ -32,7 +32,7 @@ public async Task<Guid> CreateDisputeAsync(NoticeOfDispute disputeToSubmit)
 
             using (var httpClient = new HttpClient())
             {
-                var oracleDataApiBaseUri = $"http://{_oracleDataApiConfiguration.Host}:{_oracleDataApiConfiguration.Port}/api/v1.0/";
+                var oracleDataApiBaseUri = $"{_oracleDataApiConfiguration.BaseUrl}api/v1.0/";
```

Comment:
We're not using a generated client for workflow service right now but this can be definitely updated and improved in the future. I just don't want to block the functionality to create a dispute for now.